### ISSUE TITLE
Don't charge hosts for inactive/unsuccessful Collectives

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#3392]
## User story 
As a host admin, I want to feel free to set up new Collectives and not have to pay for them on my host plan if they aren't being used.

Host admin feedback:

> Charging for only “active” collectives would make a lot of sense. This would enable us to encourage groups that we are working with to use the OC service with less of a worry about hikes in monthly fees.

> We haven’t yet selected a new plan, for a number of reasons. Not least among these is the cost involved. We currently have 11 collectives under our umbrella, which, if we opted to move away from our current position, would immediately cost us $100 per month, with no real discernible benefit. That’s not  affordable for us in any sustainable way. We are working with the collectives we host to encourage them to either make their collectives work better or shut them down. This might bring us down to the $50 per month level. Still a hefty cost for us to shoulder when I’m still struggling to see clear benefits other than the ability to avoid costly payment processors. I understand that OC needs to turn a penny, but these stepped pricing levels feel very steep, and act to disincentivise us from encouraging groups that we work with to use open collective.

## Best solution for this problem
Do not count inactive Collectives toward the total we base paid host plans on. Right now archives Collectives aren't counted, but this doesn't fix the issue of Collectives which are inactive but not archived. We would need to come up with a good definition of "active". I think a Collective with any activity in the last 30 days (change to ledger, updates, change to profile info) should be considered "active".

Another idea would be to start charging hosts for a Collective only after it brings in $X income. If the average cost to a host for a Collective is about $10/month and the average host fee is 5%, then it makes more sense to only charge them if the Collective has brought in at least $200 that month.

**Comment by macbro3** [comment]
That's great.

**Comment by patcon** [comment]
Agree with this thinking. As a creative exercise in complexity thinking, I'm going to riff about the patterns that seem to have emerged and succeeded in another organic, self-replicating system: animals! 🐴 🐷 🤸‍♀️ 

- Creation should be very accessible to hosts, maybe even incentivized. Creating new collectives should feel so good it releases endorphins.
- Hosts should be encouraged to actively nurture collectives until they can stand on their own. They should be incentivized to offer resources that protect and nurture.
- The collectives should experience lots of freedom under the host, and be able to kinda live quite freely until they find success.
- Collectives should have a fear of death. There is a lower limit under which they might fall and it not be trivial to be revived.
- Other things? I dunno, I'm sure there's more inspiration.

Of course, the above in a more mammalian perspective. Maybe hosts and collectives might want to looks more to amoebas for inspiration? or spiders! "let's make _so many collectives_, and hosts don't care, and maybe some smaller hosts get eaten" 😄

**Comment by alanna** [comment]
I loved your comment @patcon :D
